backend/compact-connect/lambdas/purchases/tests/function/test_handlers/test_privilege_options.py

```diff
@@ -0,0 +1,96 @@
+            event['requestContext']['authorizer']['claims']['custom:compact'] = test_compact
+
+        return event
+
+    def test_get_purchase_privilege_options_returns_expected_jurisdiction_option(self):
+        from handlers.privileges import get_purchase_privilege_options
+        event = self._when_testing_provider_user_event_with_custom_claims()
+
+        resp = get_purchase_privilege_options(event, self.mock_context)
+
+        self.assertEqual(200, resp['statusCode'])
+        privilege_options = json.loads(resp['body'])
+
+        with open('tests/resources/dynamo/jurisdiction.json', 'r') as f:
+            expected_jurisdiction_option = json.load(f)
+            expected_jurisdiction_option.pop('pk')
+            expected_jurisdiction_option.pop('sk')
+            # we should not be returning email information in the response
+            expected_jurisdiction_option.pop('jurisdictionOperationsTeamEmails')
+            expected_jurisdiction_option.pop('jurisdictionAdverseActionsNotificationEmails')
+            expected_jurisdiction_option.pop('jurisdictionSummaryReportNotificationEmails')
+            # remove date fields as they are not needed in the response
+            expected_jurisdiction_option.pop('dateOfUpdate')
+
+
+        # the jurisdiction configuration is stored in the dynamo db as part of the
+        # parent TstFunction setup, so we can compare the response directly
+        jurisdiction_option = [option for option in privilege_options['items'] if option['type'] == 'jurisdiction'][0]
+        self.assertEqual(expected_jurisdiction_option, jurisdiction_option)
+
+
+    def test_get_purchase_privilege_options_returns_expected_compact_option(self):
+        from handlers.privileges import get_purchase_privilege_options
+        event = self._when_testing_provider_user_event_with_custom_claims()
+
+        resp = get_purchase_privilege_options(event, self.mock_context)
+
+        self.assertEqual(200, resp['statusCode'])
+        privilege_options = json.loads(resp['body'])
+
+        with open('tests/resources/dynamo/compact.json', 'r') as f:
+            expected_compact_option = json.load(f)
+            expected_compact_option.pop('pk')
+            expected_compact_option.pop('sk')
+            # we should not be returning email information in the response
+            expected_compact_option.pop('compactOperationsTeamEmails')
+            expected_compact_option.pop('compactAdverseActionsNotificationEmails')
+            expected_compact_option.pop('compactSummaryReportNotificationEmails')
+            # remove date fields as they are not needed in the response
+            expected_compact_option.pop('dateOfUpdate')
+
+        # the compact configuration is stored in the dynamo db as part of the
+        # parent TstFunction setup, so we can compare the response directly
+        compact_option = [option for option in privilege_options['items'] if option['type'] == 'compact'][0]
```

Comment:
should we test that this list is only of length 1 before pulling out the first element?

Reply:
We can, either way the test would fail, since it will raise an index error and the developer would need to correct the issue regardless.

Reply:
Ah never mind, I misread your comment. Sure I'll add that check in.